addons/l10n_it_edi/wizard/account_move_send.py

```diff
@@ -1,12 +1,183 @@
 # -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from base64 import b64encode
+from markupsafe import Markup
+
+from odoo import _, api, fields, models, SUPERUSER_ID
+from odoo.exceptions import UserError
+from odoo.addons.l10n_it_edi.tools.xml_utils import format_errors
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
 
 
 class AccountMoveSend(models.Model):
     _inherit = 'account.move.send'
 
-    def _get_mail_attachment_from_doc(self, doc):
-        if doc.edi_format_id.code == 'fattura_pa':
-            return []
-        return super()._get_mail_attachment_from_doc(doc)
+    enable_l10n_it_edi = fields.Boolean(compute='_compute_send_mail_extra_fields')
+    l10n_it_edi_checkbox = fields.Boolean('Tax Agency (Italy)', compute='_compute_l10n_it_edi_checkbox',
+        store=True, readonly=False, help=(
+            "Send the invoice to the Italian Tax Agency.\n"
+            "It is set as readonly if a report has already been created, to avoid inconsistencies.\n"
+            "To re-enable it, delete the PDF attachment."))
+    l10n_it_edi_readonly = fields.Boolean(compute='_compute_send_mail_extra_fields')
+    l10n_it_edi_warning = fields.Html(compute='_compute_send_mail_extra_fields')
+
+    # -------------------------------------------------------------------------
+    # COMPUTE/CONSTRAINS METHODS
+    # -------------------------------------------------------------------------
+
+    @api.depends('enable_l10n_it_edi')
+    def _compute_send_mail_extra_fields(self):
```

Comment:
feel free to rename this method to `_compute_from_moves` or something with more sense ;)